edm4hep.yaml

```
@@ -403,9 +403,12 @@ datatypes :
       - edm4hep::Track                 tracks       //tracks that have been used for this particle. 
       - edm4hep::ReconstructedParticle particles    //reconstructed particles that have been combined to this particle.
       - edm4hep::ParticleID            particleIDs  //particle Ids (not sorted by their likelihood) 
+    ConstExtraCode:
```

```suggestion
    ExtraCode:
```
This is the actual category.